### ISSUE TITLE
FACT : _ASSO : _NOTE is not displayed

### ISSUE THREAD

**Issue by Joppla** [fisharebest/webtrees#1284]
The notice added to an associated person (I use it to give extra information about the associated person) isn't shown anywhere (or not anywhere logical).  It could be nice if the information is shown behind the associated person (behind the name and relation in the underneath example)
![afbeelding](https://user-images.githubusercontent.com/25467829/28248401-7509e46e-6a44-11e7-9a81-0b74af1e27fc.png)

**Comment by fisharebest** [comment]
> The notice added to an associated person (...) isn't shown anywhere

What exactly is not shown?

Can you provide some GEDCOM data that illustrates what is missing.

**Comment by Joppla** [comment]
Hi Greg,

The 3 NOTE after 3 RELA is not shown. 

1 DEAT
2 DATE 07 OCT 1855
3 TIME 08:30
2 AGE 29 jaren
2 PLAC Edam, Noord-Holland, Nederland
2 ADDR in het huis staande op de nieuwe haven, wijk 6, nummer 35, te Edam
2 SOUR @S271@
2 OBJE @M2425@
2 _ASSO @I3285@
3 RELA informant
3 NOTE aannemer, wonende op de groote bult alhier
2 _ASSO @I276@
3 RELA informant
-- 
Met vriendelijke groet,
Jop
Sent from my Android device with K-9 Mail. Please excuse my brevity.

Greg Roach <notifications@github.com> schreef op 16 juli 2017 18:20:26 CEST:

**Comment by fisharebest** [comment]
```
1 DEAT
2 _ASSO @I3285@
3 NOTE aannemer, wonende op de groote bult alhier
```

See GEDCOM spec, "association structure", p31

**Comment by Joppla** [comment]
hi Greg,
Is your comment for me or for the developers how to make a solution?

**Comment by fisharebest** [comment]
Comment for me.

**Comment by Joppla** [comment]
Am I right that this item is not solved in 1.7.10?

**Comment by fisharebest** [comment]
Fixed in the latest dev code.
